docs/whatsnew/changelog.rst

```diff
@@ -4,5 +4,7 @@
 Full Changelog
 **************
 
-.. changelog:: ../
-.. include:: ../../CHANGELOG.rst
+.. changelog::
+   :towncrier: ../
+   :towncrier-skip-if-empty:
+   :changelog_file: ../CHANGELOG.rst
```

Comment:
This will fix the build, but I want to see it error rather than timeout first.
```suggestion
   :towncrier: ../../
   :towncrier-skip-if-empty:
   :changelog_file: ../../CHANGELOG.rst
```